packages/utilities/src/webhook_payload_template.ts

```diff
@@ -125,31 +133,74 @@ export class WebhookPayloadTemplate {
     }
 
     private _parse() {
+        let currentIndex = 0;
         while (true) {
             // eslint-disable-line no-constant-condition
             try {
                 return JSON.parse(this.payload);
             } catch (err) {
-                const position = this._findPositionOfNextVariable();
-                if (position) {
-                    this._replaceVariable(position);
-                } else {
-                    // When we catch an error from JSON.parse, but there's
-                    // no variable, we must have an invalid JSON.
+                const position = this._findPositionOfNextVariable(currentIndex);
+                // When we catch an error from JSON.parse, but there's no remaining variable, we must have an invalid JSON.
+                if (!position) {
                     throw new InvalidJsonError(err as Error);
                 }
+                if (!position.isInsideString) {
+                    this._replaceVariable(position);
+                }
+                currentIndex = position.openBraceIndex + 1;
             }
         }
     }
 
-    private _findPositionOfNextVariable(startIndex = 0): { openBraceIndex: number, closeBraceIndex: number } | null {
+    private _interpolate(value: any): any {
+        if (typeof value === 'string') {
+            return this._interpolateString(value);
+        }
+        // Array needs to go before object!
+        if (Array.isArray(value)) {
+            return this._interpolateArray(value);
+        }
+        if (typeof value === 'object' && value !== null) {
+            return this._interpolateObject(value);
+        }
+        // We can't interpolate anything else
+        return value;
+    }
+
+    private _interpolateString(value: string): string {
+        // If the string matches exactly, we return the variable value including the type
+        if (value.match(/^\{\{var:([a-zA-Z0-9.]*)\}\}$/)) {
```

Comment:
This would make `{{var:}}` valid too, you should have `+` instead of `*` in the regex

Reply:
fixed